docs/source/Running_ASCENT/Info.md

```diff
@@ -379,6 +377,24 @@ upper and lower bound stimulation amplitudes were within 1%.
 Comparison of thresholds between the originally published models and the interpolation of the MRG fiber diameters (evaluated at the original diameters). Thresholds are expected to vary between the originally published models and the interpolated fiber geometries given their slightly different ultrastructure parameters. Used original MRG thresholds as reference.
 ```
 
+#### Small fiber MRG interpolated diameters - Version 1
```

Comment:
somewhere here should reference edgar paper which introduced these

Reply:
Ie change into sentences to say somewhere: "we implemented new small diameter mrg fiber model as described in [paper ref]